erica/application/JobService/job.py

```diff
@@ -46,8 +46,8 @@ async def perform_job(request_id: UUID, repository: base_repository_interface, s
                 f"Job failed: {entity}. Got Error Message: {e.generate_error_response(True).__str__()}",
                 exc_info=True
             )
-            entity.error_code = e.generate_error_response().get('code')
-            entity.error_message = e.generate_error_response().get('message')
+            entity.error_code = e.generate_error_response().get('message')
```

Comment:
A bit confusing that we use error_code for the actual "message" from erica, but we create ourselves that dict with code-message-validationproblems right? So it should be fine, but it is quite hard to read

Reply:
You are right. We should change the e.generate_error_response method I think. However, there is still some dependency with v1 that would take some time to make right. Perhaps have a separate parameter details and call the code number would help?